app/background/events/cosmeticFilterEvents.ts

```diff
@@ -0,0 +1,87 @@
+// import windowActions from '../actions/cosmeticFilterActions'
+// import tabActions from '../actions/tabActions'
+
+// import * as cosmeticFilterAPI from '../api/cosmeticFilterAPI'
+// background
+
+// this definitely goes into state
+import cosmeticFilterActions from '../actions/cosmeticFilterActions'
+
+let rule = {
+  host: '',
+  selector: ''
+}
+
+// add context menu
+chrome.runtime.onInstalled.addListener(function () {
+
+  // parent menu
+  chrome.contextMenus.create({
+    title: 'Brave',
+    id: 'brave',
+    contexts: ['all']
+  })
+
+  // block ad child menu
+  chrome.contextMenus.create({
+    title: 'Block ad',
```

Comment:
I have an opinion on these context menu titles that I'll describe in the overall review comment